src/test/java/org/mockbukkit/mockbukkit/plugin/PluginManagerMockTest.java

```diff
@@ -390,6 +392,22 @@ void loadPlugin_InvalidName_DoesntLoad(String name) throws ReflectiveOperationEx
 		assertThrows(PluginLoadException.class, () -> pluginManager.loadPlugin(TestPlugin.class, sillyName, new Object[0]));
 	}
 
+	@Test
+	void loadPluginViaPureLoadPlugin()
+	{
+		// Normal flow
+		JavaPlugin loadedPlugin = pluginManager.loadPlugin(TestPlugin.class);
+		assertInstanceOf(JavaPlugin.class, loadedPlugin);
+		assertEquals("MockBukkitTestPlugin", loadedPlugin.getName());
+		assertEquals("0.1.0", loadedPlugin.getDescription().getVersion());
+
+		// FileNotFound flow
+		loadedPlugin = pluginManager.loadPlugin(JavaPlugin.class); // Internal plugin, can't find plugin.yml file
+		assertInstanceOf(JavaPlugin.class, loadedPlugin);
+		assertEquals("JavaPlugin", loadedPlugin.getName());
+		assertEquals("0.0.0", loadedPlugin.getDescription().getVersion());
```

Comment:
Can we separate this into 2 unit tests?
1. Loading plugin with descriptor file
2. Loading plugin without descriptor file